src/components/CodeEditorWindow/menu.js

```diff
@@ -0,0 +1,159 @@
+import React, { useState } from 'react';
+import { SwipeableDrawer, Button, Box, Stack, Typography, ButtonGroup } from '@mui/material'
+
+import Editor from "@monaco-editor/react";
+
+export default function Menu({ code, handleEditorChange }) {
+  const [state, setState] = React.useState(false);
+  const [currentSavedCodes, setCurrentSavedCodes] = useState(localStorage.getItem("currentSavedCodes") ? JSON.parse(localStorage.getItem("currentSavedCodes")) : []);
+  const [viewCode, setViewCode] = React.useState("//Selected Code");
+
+  const toggleDrawer = (open) => (event) => {
+    setCurrentSavedCodes(localStorage.getItem("currentSavedCodes") ? JSON.parse(localStorage.getItem("currentSavedCodes")) : []);
+    setState(open);
+  };
+
+  function deleteSavedCode(index) {
+    currentSavedCodes.splice(index, 1);
+    localStorage.setItem("currentSavedCodes", JSON.stringify(currentSavedCodes));
+    setCurrentSavedCodes(JSON.parse(localStorage.getItem("currentSavedCodes")))
+    return;
+  }
+
+  function formatJSON(val = {}) {
+    if (val && Object.keys(val).length !== 0) {
+      try {
+        return JSON.stringify(val, null, 2)
+      } catch {
+        const errorJson = {
+          "error": `HERE ${val}`
+        }
+        return JSON.stringify(errorJson, null, 2)
+      }
+    }
+    else {
+      return ""
+    }
+  }
+  return (
+      <React.Fragment key={"History mode"}>
+        <Button onClick={toggleDrawer(true)}>History mode</Button>
```

Comment:
It looks like buttons menu will be extended, it would be useful to have all buttons grouped and aligned properly. I suggest the following:

```suggestion
<Stack 
          spacing={2}
          direction="row"
          sx={{ 
            pl: 5,
            mb: 1,
            borderBottom: (theme) => `1px solid ${theme.palette.divider}`,
          }}>
          <Button onClick={toggleDrawer(true)}>History</Button>
        </Stack>
```

Reply:
![image](https://user-images.githubusercontent.com/1935623/228367718-7e2b5596-c749-44ab-b51d-bfbf46b82a8c.png)